.github/workflows/build.yaml

```diff
@@ -375,3 +375,38 @@ jobs:
 
       - name: Run codespell
         run: codespell -q 7 -S ".git,build*,./src/third_party/*" -I misc/codespell-allowlist.txt
+
+  deploy:
+    name: Deploy
+    runs-on: windows-2019
+    strategy:
```

Comment:
It shouldn't be necessary to disable fail-fast.

Reply:
Removed superfluous code